projects/js-packages/ai-client/src/data-flow/context.tsx

```diff
@@ -61,7 +61,7 @@ type AiDataContextProviderProps = {
  *
  * @returns {AiDataContextProps} Context.
  */
-export const AiDataContext = createContext( {} as AiDataContextProps );
+export const AiDataContext = createContext( {} ) as React.Context< AiDataContextProps >;
```

Comment:
I'm curious what is the advantage of using `@wordpress/element` library, that basically re-exports untyped React functions ([source](https://github.com/WordPress/gutenberg/blob/6e9dca5947288e49834396b99006d3467b9a7e1a/packages/element/src/react.js))

If we would import `createContext()` directly from "react" library, we could do the following, no casting necessary:
```suggestion
export const AiDataContext = createContext< AiDataContextProps >( {} );
```

Reply:
That is good to know, and I suppose I should have checked before choosing which one to use. I thought the hooks coming from `@wordpress/element` were supposed to be used since they come from an internal library, but if they're untypes and predate typescript, I guess it's better to use the React ones. Thank you for this info